frontend/src/components/collaborator/CollaboratorListItem.vue

```diff
@@ -1,138 +1,88 @@
-              :disabled="disabled || isLastManager"
-            />
-          </div>
-        </template>
-        <span>{{
-          $tc(
-            'components.collaborator.collaboratorListItem.cannotAssignAnotherRoleToLastManager'
-          )
-        }}</span>
-      </v-tooltip>
-    </v-list-item-action>
-    <v-list-item-action class="ml-2">
-      <v-tooltip :disabled="disabled || !isLastManager" top>
-        <template #activator="{ on, attrs }">
-          <div v-bind="attrs" v-on="on">
-            <CollaboratorListItemDeactivate :entity="collaborator">
-              <template #activator="{ on: onDialog }">
-                <button-delete
-                  :disabled="(disabled && !isOwnCampCollaboration) || isLastManager"
-                  icon="mdi-cancel"
-                  v-on="onDialog"
-                >
-                  {{ $tc('components.collaborator.collaboratorListItem.deactivate') }}
-                </button-delete>
-              </template>
-            </CollaboratorListItemDeactivate>
-          </div>
-        </template>
-        <span>{{
-          $tc('components.collaborator.collaboratorListItem.cannotRemoveLastManager')
-        }}</span>
-      </v-tooltip>
+    <v-list-item-action class="e-collaborator-item__actions ml-2">
+      <button-edit
+        v-if="editable"
+        color="primary--text"
+        text
+        class="my-n1 v-btn--has-bg"
+      />
     </v-list-item-action>
   </v-list-item>
 </template>
 
 <script>
-import ApiSelect from '@/components/form/api/ApiSelect.vue'
 import UserAvatar from '@/components/user/UserAvatar.vue'
-import IconButton from '@/components/buttons/IconButton.vue'
-import CollaboratorListItemDeactivate from '@/components/collaborator/CollaboratorListItemDeactivate.vue'
-import { errorToMultiLineToast } from '@/components/toast/toasts'
+import ButtonEdit from '@/components/buttons/ButtonEdit.vue'
 
 export default {
   name: 'CollaboratorListItem',
   components: {
-    ApiSelect,
+    ButtonEdit,
     UserAvatar,
-    IconButton,
-    CollaboratorListItemDeactivate,
   },
   props: {
     collaborator: { type: Object, required: true },
-    disabled: { type: Boolean, default: false },
+    editable: { type: Boolean, default: false },
   },
   data: () => ({
-    resendingEmail: false,
+    roles: {
+      manager: {
+        roleTranslation: 'entity.camp.collaborators.manager',
+        abilitiesTranslation: 'global.collaborationAbilities.manager',
+        icons: ['mdi-eye-outline', 'mdi-pencil-outline', 'mdi-cog-outline'],
+      },
+      member: {
+        roleTranslation: 'entity.camp.collaborators.member',
+        abilitiesTranslation: 'global.collaborationAbilities.member',
+        icons: ['mdi-eye-outline', 'mdi-pencil-outline'],
+      },
+      guest: {
+        roleTranslation: 'entity.camp.collaborators.guest',
+        abilitiesTranslation: 'global.collaborationAbilities.guest',
+        icons: ['mdi-eye-outline'],
+      },
```

Comment:
Consider extracting these roles into a separate file and reusing them for the dropdown select

Reply:
Where would you place such a file?